Source/Particles/Filter/FilterFunctors.H

```diff
@@ -125,7 +125,7 @@ struct ParserFilter
             uz /= m_mass;
         }
         // ux, uy, uz are now in beta*gamma
-        if ( m_function_partparser(m_t,x,y,z,ux,uy,uz) ) return 1;
+        if ( m_function_partparser((amrex::ParticleReal)m_t,x,y,z,ux,uy,uz)) return 1;
```

Comment:
I actually thought about this...
```suggestion
        if ( m_function_partparser(m_t,x,y,z,ux,uy,uz)) return 1;
```